libs/cloud/odc/aws/queue.py

```diff
@@ -180,3 +181,30 @@ def get_messages(
         raise Exception(f"Limit {limit} is not valid.")
 
     return itertools.islice(messages, limit)
+
+
+def publish_to_topic(topic_name: str, action: str, stac: dict):
+    """
+    Publish 'added' or 'archived' action to the provided sns topic
+    """
+    sns = boto3.client("sns")
+    sns_topic = sns.create_topic(Name=topic_name)
+
+    stac = json.dumps(stac)
+    message = {
+        "action": action,
+        "stac": stac,
+    }
+    sns.publish(
+        TopicArn=sns_topic.get("TopicArn"),
+        Message=json.dumps(message),
+        MessageAttributes={
```

Comment:
Is it worth adding more `MessageAttributes` such as the `product` (so we can filter on it, beyond just whether it was added/archived)? Maybe `maturity` is worth adding too?

Reply:
What would the expected behaviour be for products that don't have `maturity`?

Reply:
Not sure, haven't come across that. I'd guess don't pass it as a `MessageAttribute` if it's not in the STAC document.